.github/workflows/nightly-image-build.yaml

```diff
@@ -0,0 +1,50 @@
+name: Nightly Image Build
+
+on:
+    workflow_dispatch:
+    schedule:
```

Comment:
Can you rather use `push` event (i.e. https://github.com/project-codeflare/codeflare-operator/blob/main/.github/workflows/operator-image.yml#L6)?
That way the `dev` image is refreshed immediately.